packages/ui/components/pages/Fuse/FusePoolsPage/FusePoolList/index.tsx

```diff
@@ -26,7 +66,15 @@ const FusePoolList = () => {
     );
   }
 
-  return <PoolsRowList poolsPerChain={poolsPerChain} isLoading={isLoading} />;
+  return preSearchText !== undefined && preFilter !== undefined && preSorting !== undefined ? (
```

Comment:
Feels odd. That these values need to be defined. What if local storage is empty?

If these have not `undefined` initial values, we should set them directly in the `useState('')` hook instead. No extra stuff needed in line: 43

`_initSearch = _initSearch ? _initSearch : '';`

Also `PoolsRowList` could have default values defined in it's props for these if they could be undefined.

Reply:
nvm, fixed